src/Kinto.elm

```diff
@@ -665,10 +650,10 @@ toRequest builder =
 
 -}
 get : Resource a -> String -> Client -> Request a
-get resource itemId client =
-    endpointUrl client.baseUrl (resource.itemEndpoint itemId)
```

Comment:
I don't understand the use for client**instance** everywhere, does it make it more clear?